app/templates/learnergroups.hbs

```diff
@@ -40,11 +46,11 @@
       <div class="resultslist-title">
         <h2>{{title}}</h2>
       </div>
-      {{#if currenCohort}}
+    {{!-- {{#if currentCohort}} --}}
```

Comment:
It is possible that there are NO cohorts in a program.  Or no programs in a school.  Or no schools in the system.  So while if there is at least one cohort we do force the selection of it, we still have to be wary of situations where there isn't.